convention.md

```diff
@@ -475,3 +484,9 @@ homie/$broadcast/alert ← "Intruder detected"
 ```
 
 Any other topic is not part of the Homie convention.
+
+### Optional Extensions
+
+In addition to homie-core the device may support multiple extensions which are defined in separate conventions.
+Every extension is identified by a unique id, which is similar to a topic id but also allows dots.
+The id consists of the reverse domain name of a vendor owned domain and a freely choosen prefix. For example a company owning `example.org` that likes likes to extend homie a cool feature would choose `org.example.cool-feature`. If it is a official extension the prefix must be `homie`.
```

Comment:
You don't need to own a domain to choose a unique reverse domain name and you can't make this mandatory in my opinion, if homie itself doesn't own its own reverse domain name namespace.

> If it is a official extension the prefix must be `homie`.

That sounds more like an internal instruction. The wording in other specifications is more like "The namespace x.y is reserved."